### PR TITLE
allow account names starting with a number

### LINKED ISSUE #72: support Beancount accounts starting with a digit
Beancount [now allows account names starting with digits](https://bitbucket.org/blais/beancount/issues/245/why-is-account-name-starting-with-a-digit).

* Update the converter.
* Document supported beancount version?

### PR DESCRIPTION
**Pull request by tbm**
As of beancount 2.0.0, account names are allowed to start with a
number.

Fixes #72

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,6 @@ python:
 install:
   - sudo apt-get install libcarp-assert-perl libdate-calc-perl libyaml-libyaml-perl libfile-basedir-perl libexperimental-perl libgetopt-long-descriptive-perl cpanminus
   - sudo cpanm Config::Onion
-  - pip install beancount==2.0rc1
+  - pip install beancount==2.0
```
Comment (zacchiro) [inline .travis.yml]: This should go in a separate commit

Comment (tbm) [reply .travis.yml]: done
